### ISSUE TITLE
effect_outline generates large blobs of color

### ISSUE THREAD

**Issue by clach04** [ygalanter/EffectLayer#17]
if orig_color and offset_color are set to the same color. E.g. tweak to documentation example:

    EffectOffset shadow;
    shadow.orig_color = GColorRed;
    shadow.offset_color = shadow.orig_color;  // For a text layer, attempt to make it bold
    shadow.offset_y = 2;  
    shadow.offset_x = 2;

    effect_layer_add_effect(my_effect_layer, effect_outline, &shadow); - Similar to the Shadow effect, however the color is added in all directions.

**Comment by clach04** [comment]
I have a zip of a test case which I can email - github only allows images to be attached :-(

**Comment by ygalanter** [comment]
Thanks for reporting the issue. Actually I think this is behavior by design (though effect is unintended). This effect uses `orig_color` as a starting border to draw the outline in `offset_color`. If both are the same - starting border will grow exponentially - resulting in the blob. You may want to contact original contributor of the effect (https://github.com/m6ao) if you have a different usecase. But currently, for example, if you want a shape or a text to become bolder - use similar very close colors, for example `GColorDukeBlue` and `GColorOxfordBlue`

**Comment by clach04** [comment]
I wasn't sure if it was by design or not, that was my main reason for opening the issue :-D

@m6ao any comments? For Basalt the workaround suggested by @ygalanter  looks like it will work perfectly. For Aplite I don't think that is an option. I don't feel strongly about this so if you tell me it is by design, that is OK by me but this should be documented (ideally in comments and the readme).

I did wonder if the aplite_visited[] was useful here? The readme indicates this is only useful when using "...option = 1; to display long contiuous [sic.] shadow"

**Comment by ygalanter** [comment]
Yes I implemented “aplite_visited[]” as Aplite workaround for inability to use color as border (and used it in 2 faces “Long Shadow” and “Shadow Analog”) so I guess something similar could be used in Outline. But that workaround feels hacky, maybe there’s a better way to implement outline on Aplite (which technically would mean simple make original shape thicker)

**Comment by clach04** [comment]
I've sent m6ao an email to get his opinion. I'm not sure if the "@" syntax in the message causes a notification.
